bbc/simorgh
# WSTEAM1-457 - `.app` route ATI param changes

Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAM1-457

Overall changes
======
- Updates of the ATI tracking params for `.app` routes to help identify them

Code changes
======
- Adds `app` as a `Platform` type
- Updates `buildATIPageTrackPath` to account for the `platform` and `appName` when setting the `x2` and x3` values

Testing
======
1.  Visit http://localhost:7080/kyrgyz/articles/ck7v78zvng9o?renderer_env=live
2. Open the ATI Tag Inspector and check that `#2 = responsive` and `#3 - news-kyrgyz`
3. Visit http://localhost:7080/kyrgyz/articles/ck7v78zvng9o.amp?renderer_env=live
4. Open the ATI Tag Inspector and check that `#2 = amp` and `#3 - news-kyrgyz`
5. Visit http://localhost:7080/kyrgyz/articles/ck7v78zvng9o.app?renderer_env=live
6. Open the ATI Tag Inspector and check that `#2 = mobile-app` and `#3 - news-kyrgyz-app`

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
